src/server/MongooseModels/regist.ts

```diff
@@ -0,0 +1,22 @@
+
+import mongoose from 'mongoose'
+
+const RegistSchema = new mongoose.Schema({
+  vaseId: {
+    type: Number,
+    required: true
+  },
+  country: {
+    type: String
+  },
+  city: {
+    type: String
+  },
+  text: {
+    type: String
+  }
+}, { timestamps: true })
```

Comment:
I didnt know this, but just checked this out, and this enables createdAt and updatedAt, rigth? 

We will not need updatedAt because each insert will not be updated, so maybe just set the timestamp as:
```
timestamps:{createdAt:true, updatedAt:false}
```

Reply:
Yes, this adds both createdAt and updatedAt.
Timestamp will now only show createdAt